components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
```

Comment:
despite the fact that doing https://github.com/manics/openmicroscopy/commit/a9d87d9d8bf1a3ad04e89a7137a55823f9726ce5#diff-e226bce909b5b71ca51f5a26c7db1823R748 is dangerous!